lib/src/create_report.dart

```diff
@@ -668,6 +663,9 @@ Future<ReportSection> _multiPlatform(String packageDir, Pubspec pubspec) async {
       }
     } else {
       tagger.runtimeTags(tags, explanations);
+      final issues = explanations
```

Comment:
not sure, just double-checking: is this `issues` masking another `issues` variable in score?

Reply:
This was accidentally mixed in from https://github.com/dart-lang/pana/pull/689 and doesn't belong here.

Thanks!